apps/fm/index.html

```diff
@@ -41,13 +41,13 @@
       </div>
       <div id="action-bar">
         <div>
-          <a id="frequency-op-seekdown" href="#seekdown"></a>
+          <a id="frequency-op-seekdown" href="#seekdown" data-l10n-id="frequency-op-seekdown" role="button"></a>
         </div>
         <div>
-          <a id="power-switch" href="#power-switch" data-enabled="false" data-enabling="false"><span></span></a>
+          <a id="power-switch" href="#power-switch" data-enabled="false" data-enabling="false" data-l10n-id="power-switch" role="button"><span></span></a>
```

Comment:
This button should have its labelled changed depending on whether it is an On or Off button. So we need to update data-l10n-id dynamically at the same time the icon is updated.